paypal/merchant-sdk-php/CVE-2017-6099.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Cross-site scripting (XSS) vulnerability in Paypal-Merchant-SDK-PHP
+link:      https://github.com/paypal/merchant-sdk-php/tree/v3.9.1
+cve:       CVE-2017-6099
+branches:
+    2.x:
+        time:     ~,
```

Comment:
comma should be removed

Reply:
Does that mean that there is no fix for this issue?

Reply:
I put no time for that branch because the issue was not fixed for that branch, but it was fixed in the master branch.